programs/clearing_house/src/state/market.rs

```diff
@@ -108,13 +108,19 @@ pub struct AMM {
     pub quote_asset_amount_long: u128,
     pub quote_asset_amount_short: u128,
 
+    // lp stuff
+    pub cumulative_funding_payment_per_lp: i128,
+    pub cumulative_fee_per_lp: u128,
+    pub cumulative_net_base_asset_amount_per_lp: i128,
+    pub amm_lp_shares: u128,
+    pub lp_cooldown_time: i64,
```

Comment:
i think we're going to need an admin function to update this

Reply:
added